src/hott/02-homotopies.rzk.md

```diff
@@ -41,10 +41,58 @@ This is a literate `rzk` file:
 Homotopy composition is defined in diagrammatic order like `#!rzk concat` but
 unlike composition.
 
+The following is the unit for compositions of homotopies.
+
+```rzk
+#def refl-htpy
+  (f : A → B)
+  : homotopy f f
+  := \ x → refl
+```
+
 ```rzk
 #end homotopies
 ```
 
+There is also a dependent version of homotopy.
+
+```rzk
+#def dependent-homotopy
+  ( A : U)
+  ( B : A → U)
+  (f g : (a : A) → B a)
+  : U
+  := (a : A) → (f a = g a)
```

Comment:
What you call a dependent homotopy here I would call just a homotopy. The notion of dependent homotopy should generalize dependent identifications. You can find a formalization of it here:

https://github.com/UniMath/agda-unimath/blob/591dd732c2bf6876871b941e952f88364c43ffe1/src/foundation/dependent-homotopies.lagda.md

Reply:
Indeed! But in rzk currently homotopies are not defined for dependent functions.

Reply:
@fredrik-bakke  Should I be the one to apply these changes globally? I'm happy to, but wanted to make sure.

Reply:
That would be great, if you're comfortable with that

Reply:
Yup! Absolutely.